catalogue/webapp/pages/works.js

```diff
@@ -412,21 +412,38 @@ const Works = ({ works }: Props) => {
         )}
 
         {works && works.results.length === 0 && (
-          <VerticalSpace size="l" properties={['padding-top']}>
+          <VerticalSpace
+            size="xl"
+            properties={['padding-top', 'padding-bottom']}
+          >
             <div className="container">
               <div className="grid">
                 <div className={grid({ s: 12, m: 10, l: 8, xl: 8 })}>
-                  <p className="h1">
+                  <p className={font('hnl', 2)}>
                     We couldn{`'`}t find anything that matched{' '}
                     <span
                       className={classNames({
-                        [font('hnl', 2)]: true,
+                        [font('hnm', 2)]: true,
                       })}
                       style={{ fontWeight: '400' }}
                     >
                       {query}
                     </span>
-                    .
+                    {workType && (
+                      <>
+                        {' '}
+                        in{' '}
+                        <span className={font('hnm', 2)}>
+                          {(workType.includes('k') && 'pictures') ||
```

Comment:
Is this a bit gross? Feels a bit gross to me. Unsure how to do it better.

Reply:
We should 100% be looking at redoing the metadata on the works.
@Heesoomoon @jennpb do we know where the `workType` remapping work is at?

Obviously this can go in, but it's not massively sustainable.

Reply:
We have some unresolved `workType`s that I'd like to get more research on before we make any changes.

Reply:
@davidpmccormick as we do something similar with creating the tabs, mayeb we can just store a map of
```JS
const workTypeLabels = { 'pictures': ['k', 'q'], 'books': ['a'] } /// blah blah
```